bundler/lib/bundler/cli.rb

```diff
@@ -509,6 +509,7 @@ def exec(*args)
     subcommand "config", Config
 
     desc "open GEM", "Opens the source directory of the given bundled gem"
+    method_option "path", :type => :string, :banner => "Open relative path of the gem source."
```

Comment:
Minor thing, but Thor currently passes `path="path"` if you specify just `--path`, which is probably never what you want 🤷‍♂️.

As a workaround, we can pass `:lazy_default => ""`, and then check this in the command and raise a missing value error?

Reply:
Makes sense, will check this